tests/test_project/models.py

```diff
@@ -43,3 +43,22 @@ class RadiusAccounting(models.Model):
                                 db_index=True,
                                 null=True,
                                 blank=True)
+
+
+class Id(models.Model):
+    id = models.BigAutoField(primary_key=True)
+    name = models.CharField(max_length=64,
+                            null=True,
+                            blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Operator(models.Model):
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
+    operator = models.ForeignKey(Id, on_delete=models.CASCADE, blank=True)
```

Comment:
this field should be named as the model used in the foreign key, so if you rename that model to `Project`, this field should be called `project`.

This is more for clean up and to improve readability of the test models (which is still important in the grand scheme of things, as complexity increases, these things matter too).